### PR TITLE
[ Tests ] Add new unit test for ReponseHandlerFilter

### PR DESCRIPTION
**Pull request by mattisonchao**
### Motivation

Add test.

### Modifications

- Add new unit test for ReponseHandlerFilter

### CODE REVIEW

**Review by codelipenghui — APPROVED** [review]

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/web/ResponseHandlerFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker.web;
+
+import static org.mockito.Mockito.*;
+import static org.testng.Assert.*;
```
Comment (codelipenghui) [inline pulsar-broker/src/test/java/org/apache/pulsar/broker/web/ResponseHandlerFilterTest.java:22]: Please avoid using star import.

Comment (mattisonchao) [reply pulsar-broker/src/test/java/org/apache/pulsar/broker/web/ResponseHandlerFilterTest.java:22]: Okay, I will fix it ASAP

**Comment by mattisonchao** [comment]
@gaoran10  PTAL

**Comment by gaoran10** [comment]
I'm not sure why we need this unit test. The unit tests in Pulsar may be enough? Or we could move the Pulsar tests to here?

**Pull request closed without merging**